stack-deploy.sh

```diff
@@ -11,6 +11,8 @@ elif [ ! -f "${HOME}/.docker/config.json" ]; then
   ln -sfn /auth.json "${HOME}/.docker/config.json"
 fi
 
+export time_prefix="${STACK_NAME}-$(date +%s)"
```

Comment:
I really have no idea what is going on here :) but is this being used anywhere?

Reply:
It’s taking the stack name, appending a Unix timestamp, and making it available to the `docker deploy stack` commands. Then we can use `$time_prefix` in the docker-stack file to create/isolate volumes to a single deploy. 

I guess a good way to look at it is cache busting for Docker volumes.